components/Dialogs/src/MDCDialogPresentationController.m

```diff
@@ -17,29 +17,36 @@
 #import "MDCDialogPresentationController.h"
 
 #import "MaterialKeyboardWatcher.h"
+#import "MDCDialogTransition.h"
```

Comment:
Nit: Same order-of-imports request here.

Reply:
Done.